factory/.env

```diff
@@ -11,7 +11,7 @@ BASE_IMAGE='debian:bullseye-slim'
 NODE_VERSION='18.14.1'
 
 # Update this to deploy the docker factory if you make changes to factory.Dockerfile or install scripts
-FACTORY_VERSION='2.0.2'
+FACTORY_VERSION='2.0.3'
```

Comment:
@AlanGreene Can we add a changelog entry to the factory changelog? https://github.com/cypress-io/cypress-docker-images/blob/master/factory/CHANGELOG.md

Reply:
Done, thanks